cfpb/cfgov-refresh
# Update for Capital Framework v5.2.0

Updates the project to use the latest release from Capital Framework.

## Changes

- Updates cf-forms to 5.1.0
- Updates cf-icons to 4.4.1

## Testing

1. run `npm install`
1. run `npm run gulp build`
1. run `./runserver.sh`
1. Navigate to `http://localhost:8000/owning-a-home/explore-rates/` the select elems should be 35px tall.
1. There are currently no inline errors on cfgov-refresh, so there's no way to test that change.

## Screenshots

__before__

<img width="365" alt="screen shot 2018-07-11 at 9 55 52 am" src="https://user-images.githubusercontent.com/1280430/42580778-ecc6b4e2-84f0-11e8-82cd-62f318d025dd.png">

__after__

<img width="303" alt="screen shot 2018-07-11 at 9 55 56 am" src="https://user-images.githubusercontent.com/1280430/42580769-ea423c0a-84f0-11e8-89b7-35efb594f84e.png">

## Checklist


## Testing checklist

### Browsers
